### PR TITLE
[MM-42385] Set title to popout window

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR sets a more descriptive title to the pop-out window to be more easily identifiable.
Title format is `Call - $channel_display_name`. In case of DM, the username of the other participant is used instead.

/cc @aaronrothschild 

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-42385

### CODE REVIEW

File: webapp/src/index.tsx [hunk 1]
```diff
@@ -413,10 +422,23 @@ export default class Plugin {
...
+            }
+
             const roles = getMyRoles(store.getState());
             const cms = getMyChannelMemberships(store.getState());
 
+            if (isDMChannel(channel)) {
+                const otherID = getUserIdFromDM(channel.name, getCurrentUserId(store.getState()));
+                const dmUser = getUser(store.getState(), otherID);
+                if (!dmUser) {
+                    await getProfilesByIdsAction([otherID])(store.dispatch as any, store.getState);
```
Comment (streamer45) [inline webapp/src/index.tsx:438]: Not a fan of using `any` but I hit this issue before and my still fairly limited Typescript/React knowledge couldn't get to an easy solution.

Error was the following:

```
src/index.tsx:438:61 - error TS2345: Argument of type 'Dispatch<AnyAction>' is not assignable to parameter of type 'DispatchFunc'.
  Types of parameters 'action' and 'action' are incompatible.
    Type 'Action' is not assignable to type 'AnyAction'.
      Property 'type' is missing in type 'ActionFunc' but required in type 'AnyAction'.

438                     await getProfilesByIdsAction([otherID])(store.dispatch, store.getState);
                                                                ~~~~~~~~~~~~~~

  node_modules/redux/index.d.ts:19:3
    19   type: T
         ~~~~
    'type' is declared here.
```

Comment (cpoile) [reply webapp/src/index.tsx:438]: Do you mind if I push a commit? It will add `redux-thunk` to the dev dependencies and simplify this a bit?

Comment (streamer45) [reply webapp/src/index.tsx:438]: By all means :raised_hands:

**Review by cpoile — APPROVED** [review]
Looks great!

**Pull request merged**